src/ophyd_async/fastcs/panda/_table.py

```diff
@@ -51,13 +51,7 @@ class SeqTrigger(str, Enum):
     ),
     Field(default_factory=lambda: np.array([], dtype=np.bool_)),
 ]
-TriggerStr = Annotated[
-    np.ndarray[tuple[int], np.unicode_],
-    NpArrayPydanticAnnotation.factory(
-        data_type=np.unicode_, dimensions=1, strict_data_typing=False
-    ),
-    Field(default_factory=lambda: np.array([], dtype=np.dtype("<U32"))),
-]
+TriggerStr = Annotated[list[SeqTrigger], Field(default_factory=list)]
```

Comment:
We've used `Sequence` rather than `list` in other places because it's [covariant rather than invariant](https://mypy.readthedocs.io/en/stable/common_issues.html#invariance-vs-covariance)

```suggestion
TriggerStr = Annotated[Sequence[SeqTrigger], Field(default_factory=list)]
```